source/opt/inline_pass.cpp

```diff
@@ -291,13 +290,9 @@ bool InlinePass::GenInlineCode(
   // at the end of the first block.  First determine if the caller is in a
   // single block loop.  We'll wait to move the OpLoopMerge until the end
   // of the regular inlining logic, and only if necessary.
-  bool caller_is_single_block_loop = false;
```

Comment:
The comment immediately above this calls out the single-block loop case.
But what's really salient about the situation is that the caller is in the loop header, and the callee has multiple blocks.
You should update the comment to reflect the more accurate/salient aspect of the situation.